services/ConfigService/tests/ConfigService.spec.ts

```diff
@@ -2,12 +2,21 @@ import { Config, NearConfig } from 'types/config';
 
 import { configService } from 'services/ConfigService';
 
+jest.mock('services/ConfigService', () => {
```

Comment:
This test makes no sense as we verify that mock works. 
I consider we should either create the service with constructor and then test the `get` method or we should remove the test at all.